.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI Micro XRCE-DDS Suite
+
+on:
+    pull_request:
+      branches:
+        - '**'
+
+jobs:
+
+  micro_xrce_dds_suite_ci:
+        runs-on: ubuntu-20.04
```

Comment:
Can you run this on 22.04 for Humble instead?

Reply:
Done, let's see if it pass CI